go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
+	github.com/carolynvs/magex v0.5.0
```

Comment:
Is there a reason this isn't in a tools module?
Putting this in the root module makes it transitive for library users.

Reply:
Also for binary only dependencies https://golang.org/doc/go1.16#go-command go1.16 go install does the right thing here.

Reply:
@BenTheElder -- I might be misunderstanding the request...

@carolynvs's `magex` extends `mage` and `sigs.k8s.io/release-utils/mage` is a package I'd plan on importing elsewhere in RelEng to remove some of the Makefile/verify/test scripts.

It's pretty small: https://github.com/carolynvs/magex/blob/abcb4ec603b4e0f4c2abc92f8978e3c208b76687/go.mod#L5-L9

Reply:
that package is small, mage is larger, there are library users that are just using the logging package here?
see for example: https://github.com/kubernetes/klog/pull/213